spacy/tests/pipeline/test_entity_linker.py

```diff
@@ -1172,3 +1173,79 @@ def create_kb(vocab):
 
     assert len(doc.ents) == 1
     assert doc.ents[0].kb_id_ == entity_id if meet_threshold else EntityLinker.NIL
+
+
+def test_store_activations():
```

Comment:
This feels like mainly copy/paste from another test, is that right? If so, can we expand on the other one instead?

Reply:
I am a bit two-minded about that. On the one hand it's less redundant, on the other hand, it's nice when you get something like

```
FAILED spacy/tests/pipeline/test_entity_linker::test_save_activations
```

because you get a nice semantic description of what failed. Getting a `FAILED ... test_overfitting_IO` is not so informative when the activation code broke.

I guess the shared code could be factored out to a separate function, but maybe that's too much?

Reply:
I agree that it's generally nice to have a single test for a single functionality and to immediately understand which functionality exactly broke when a test fails. But we want to make the trade-off against running time of the test suite. Like, I don't care so much about actually copy/pasting code in the test suite, but we do want to be mindful of blowing up the test suite too much.

Though now that we run the slow tests more regularly, we can also move more stuff to `slow`.

Reply:
Ah, right, I didn't consider running time (the downside of having fast local machines). Yeah, it'd probably make sense to mark these activation saving tests that do optimization as `slow`.

Reply:
Oh, I just realized that we don't actually need to train the entity linker in this test. So, I removed the optimization loop.